app/options.py

```diff
@@ -0,0 +1,23 @@
+from tornado.options import define, parse_config_file
+
+
+__all__ = ["load_conf", ]
+
+
+def load_conf(config_file=None):
+    # Todo: provide docstring
+    if config_file:
+        parse_config_file(config_file)
```

Comment:
I'm not sure, but first you probably need to `define` those config variables, and then parse them